docs/windows_installation.md

```diff
@@ -5,6 +5,20 @@ We **strongly** recommend that Windows users use [Docker](docker_quickstart.md)
 If that is not possible, try using the Windows Linux subsystem (WSL) - for which the Ubuntu instructions should work.
 Otherwise, please follow the instructions below.
 
+## Install freqtrade automatically
+
+### Using Invoke-WebRequest
+
+```powershell
+Invoke-WebRequest -Uri "https://raw.githubusercontent.com/freqtrade/freqtrade/stable/setup.ps1" -UseBasicParsing | Invoke-Expression
+```
+
+### Or using curl
+
+```powershell
+curl -sSL "https://raw.githubusercontent.com/freqtrade/freqtrade/stable/setup.ps1" | powershell -c -
+```
+
```

Comment:
You didn't test this, did you?

While i don't mind AI generated code necessarily - i do expect the contributor to ensure that it's actually working - which this is not (not only because of the non-working URL due to the PR not being merged).

Please spend time thinking about what the AI gave you - and if it makes sense. 
Otherwise you're wasting my time.

Reply:
I'll do that in the future. Sorry for that. If that should happen again, you can just ignore my PRs in the future. The last thing I want is wasting your time.